public/sitemap.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
```

Comment:
The `<url>` tag in sitemap.xml file accept only one loc not multiple, so please divide it to multiple `urls`.

Reply:
Thanks for the comment. I fixed them.